go.mod

```diff
@@ -5,8 +5,8 @@ go 1.17
 require (
 	github.com/fsnotify/fsnotify v1.5.1
 	github.com/opencontainers/runc v1.1.2
-	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
-	github.com/opencontainers/runtime-tools v0.0.0-20190417131837-cd1349b7c47e
+	github.com/opencontainers/runtime-spec v1.0.3-0.20220825212826-86290f6a00fb
```

Comment:
Question: I don't see `v1.0.3` or `v0.9.1` tags in these repos. Is this because these are pseudo versions that gomod bumps to the next patch release?

Reply:
Runtime tools has been going now for years untagged. We have to hand-pick/lock its version at or past the two commits that prevent the xeipuuv/jsonschema dependencies from always getting pulled in downstream. That sha1 is the tip of that tree. Same story for runtime-spec wrt. tagging. Last tagged one (1.0.2) is from January 2020. We don't need the absolute latest version, but we need newer than 1.0.2.